segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/SystemJournal.java

```diff
@@ -228,7 +228,7 @@ public SystemJournal(int containerId, ChunkStorage chunkStorage, ChunkMetadataSt
         this.garbageCollector = Preconditions.checkNotNull(garbageCollector, "garbageCollector");
         this.containerId = containerId;
         this.systemSegments = getChunkStorageSystemSegments(containerId);
-        this.systemSegmentsPrefix = NameUtils.INTERNAL_SCOPE_NAME;
+        this.systemSegmentsPrefix = NameUtils.INTERNAL_CONTAINER_PREFIX;
```

Comment:
Is this a backward compatible change? We're changing this from `_system` to `_system/containers`

Reply:
I see below that this is mostly an idempotent change; how is `this.systemSegmentsPrefix` used though?

Reply:
This just replaces "_system" + /containers/metadata...." with "_system/containers/" + "metadata....".
No functional change.

This method was actually not used anywhere in the production code until now. (i don't remember what was it used for when I first wrote that :) 
Not yet sure if this is good enough or we need to do something more fancy like regex etc. There are 4 fixed names per containers if segment is one of those it's a system segment.
```
    /**
     * Indicates whether given segment is a system segment.
     *
     * @param segmentName Name of the segment to check.
     * @return True if given segment is a system segment.
     */
    public boolean isStorageSystemSegment(String segmentName) {
        if (segmentName.startsWith(systemSegmentsPrefix)) {
            for (String systemSegment : systemSegments) {
                if (segmentName.equals(systemSegment)) {
                    return true;
                }
            }
        }
        return false;
    }
```